backend/src/db-utils.ts

```diff
@@ -748,12 +788,12 @@ const queryAccountsList = async (
   );
 };
 
-async function queryOutcomeTransactionsCountFromAnalytics(
+export const queryOutcomeTransactionsCountFromAnalytics = async (
```

Comment:
I wonder why do you prefer const + anonymous function over a named function. Is it just a code-style?

P.S. I am fine with it

Reply:
Yep, it's just a code style, there's no other point for doing that, but I do make things consistent here and there before I'll start full-blown linting.